server/routes/index.ts

```diff
@@ -1,16 +1,25 @@
 import { type RequestHandler, Router } from 'express'
-
+import ServiceCatalogueClient from '../data/serviceCatalogueClient'
 import asyncMiddleware from '../middleware/asyncMiddleware'
 import type { Services } from '../services'
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export default function routes(service: Services): Router {
   const router = Router()
   const get = (path: string | string[], handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
-
+  const catalogueclient = new ServiceCatalogueClient()
   get('/', (req, res, next) => {
     res.render('pages/index')
   })
 
+  get('/serviceselection', (req, res, next) => {
```

Comment:
This logic should be moved into a ServicesController (and we'll need supertests around it), but can maybe be done as part of the next ticket!